MIPs/mip-5.md

```diff
@@ -166,10 +178,12 @@ The CAIPs referenced in this proposal are still in Draft or Review status with C
 ### API Access
 As part of the Multichain API, MetaMask is leading the adoption of alternative methods for browser extension wallet communication. Currently, browser extension wallets inject a JavaScript object to communicate with each site. But this approach comes with privacy, security, reliability, and performance drawbacks.
 
+Wallet discovery and access will be guided by [CAIP-282]() and [CAIP-294]() standards.
+
 - The MetaMask Browser Extension will adopt an alternative method of communication through the [externally_connectable](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/externally_connectable) feature, which is now supported by most modern web browsers with extension frameworks.
-- When a site is accessed through a browser that does not support [externally_connectable](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/externally_connectable), the Multichain API will be available through an alternative communication method. 
+- When a site is accessed through a browser that does not support [externally_connectable](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/externally_connectable), the Multichain API will be available through the `window.postMessage` method. 
```

Comment:
technically MV2 supports externally_connectabel, just no wildcarded urls which makes it not usable as a wallet. Do we need to be more specific here to avoid confusion?